mobile/mobile.go

```diff
@@ -122,6 +122,16 @@ func MigrateRepo(config *MigrateConfig) error {
 func NewTextile(config *RunConfig, messenger Messenger) (*Mobile, error) {
 	node, err := core.NewTextile(core.RunConfig{
 		RepoPath: config.RepoPath,
+		LogLevels: map[string]string{
```

Comment:
sweet! just fyi: they all default to error, so you really only need `"tex-service":     "DEBUG"` here...

Reply:
oh yeah... good catch. i just c/p'd from my cli output